recipes/wg21-linear_algebra/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.12)
+project(test_package CXX)
+
+find_package(wg21_linear_algebra REQUIRED CONFIG)
```

we lost somewher ealong the way :(

```suggestion
include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

find_package(wg21_linear_algebra REQUIRED CONFIG)
```